ui/src/app/stores/workflows/workflows.effects.ts

```diff
@@ -198,4 +198,25 @@ export class WorkflowsEffects {
       );
     }),
   );
+
+  @Effect({ dispatch: true })
+  workflowRun = this.actions.pipe(
```

Comment:
this is nice, but it doesn't work.
Try run workflow and block request and do it 2 times. Second time it will stuck. 
This will work: 
`  workflowRun = this.actions.pipe(
    ofType(WorkflowActions.RUN_WORKFLOW),
    switchMap((action: WorkflowActions.RunWorkflow) => {
      return this.workflowService.runWorkflow(action.payload).pipe(
        mergeMap((runWorkflowSuccess) => {
          if (runWorkflowSuccess) {
            this.toastrService.success(texts.RUN_WORKFLOW_SUCCESS_NOTIFICATION);
            return [{ type: WorkflowActions.RUN_WORKFLOW_SUCCESS }];
          } else {
            this.toastrService.error(texts.RUN_WORKFLOW_FAILURE_NOTIFICATION);
            return [{ type: WorkflowActions.RUN_WORKFLOW_FAILURE }];
          }
        }),
        catchError(() => {
          this.toastrService.error(texts.RUN_WORKFLOW_FAILURE_NOTIFICATION);
          return [{ type: WorkflowActions.RUN_WORKFLOW_FAILURE }];
        })
      )}),
  );`